src/components/input-dropdown/input-dropdown.component.ts

```diff
@@ -53,13 +38,32 @@ export class InputDropdownComponent<T> implements ControlValueAccessor, OnChange
 
     /** Aria label of the filter field. If not specified, the placeholder will be used. */
     @Input('aria-label') ariaLabel: string = '';
-    
+
     /** Emit when the selected item is changed */
     @Output() selectedChange = new EventEmitter<T>();
 
     /** Emit when the filter text is changed */
     @Output() filterChange = new EventEmitter<string>();
 
+    /** Emits when `dropdownOpen` changes. */
+    @Output() dropdownOpenChange = new EventEmitter<boolean>();
+
+    /** The status of the dropdown. */
+    @Input()
+    set dropdownOpen(value: boolean) {
+
+        this._dropdownOpen = value;
+        this.dropdownOpenChange.emit(value);
```

Comment:
I think this is the problem that Jonathan is referring to. `emit` should only be called when the value changes programmatically?